content/applications/finance/payment_acquirers/paypal.rst

```diff
@@ -1,109 +1,86 @@
-token by following the configuration step :ref:`paypal/enable-pdt`.
-
-.. important::
-   If you are trying Paypal as a test, using a :ref:`Paypal Sandbox account <paypal/testing>`,
-   change the **State** to *Test Mode*. We recommend doing this on a test Odoo database rather than
-   on your main database.
+Settings in PayPal
+==================
 
-Fees tab
---------
+.. _paypal/auto-return:
 
-You can charge extra fees to your customers for paying with Paypal to cover the transaction fees
-Paypal charges you. Once redirected to Paypal, your customer sees an extra amount applied to the
-order amount.
+Auto Return
+-----------
 
-To activate this, go to Paypal configuration's :guilabel:`Fees` tab in Odoo and activate
-:guilabel:`Add Extra Fees`.
+The **Auto Return** feature automatically redirects customers to Odoo once the payment is processed.
 
-You can refer to `Paypal Fees <https://www.paypal.com/webapps/mpp/paypal-fees>`_ to set up fees.
+From :guilabel:`Website payments`, go to :menuselection:`Website preferences --> Update --> Auto
+return for website payments --> Auto return` and select :guilabel:`On`. Enter the address of your
+Odoo database (e.g., `https://yourcompany.odoo.com`) in the :guilabel:`Return URL` field, and
+:guilabel:`Save`.
 
 .. note::
-   `Traders in the EU <https://europa.eu/youreurope/citizens/consumers/shopping/pricing-payments/
-   index_en.htm>`_ are not allowed to charge extra fees for paying with credit cards.
+   - Any URL will do the job. Odoo only needs the setting to be enabled since it uses another URL.
+   - Turning :guilabel:`Auto Return` *off* disables the :ref:`Payment Data Transfer
+     <paypal/pdt>` feature.
 
-Settings in Paypal
-==================
-
-First, set up your Paypal account to build a seamless customer experience with Odoo.
+.. _paypal/pdt:
 
-Log into your PayPal account and open the account settings. Then, go to :menuselection:`Account menu
---> Account settings --> Website payments`.
+Payment Data Transfer (PDT)
+---------------------------
 
-Enable Auto Return
-------------------
+:guilabel:`Payment Data Transfer` allows to receive payment confirmations, displays the payment
+status to the customers, and verifies the authenticity of the payments. :abbr:`PDT (Payment Data
+Transfer)`. From :menuselection:`Website preferences --> Update`, scroll down to :guilabel:`Payment
+data transfer` and select :guilabel:`On` (if desired). **PDT** is *not* mandatory, and :ref:`IPN
+(Instant Payment Notifications) <paypal/ipn>` can be used instead.
 
-The *Auto Return* feature automatically redirects your customers to Odoo once the payment is
-processed.
+.. note::
+   To use **PDT**, :ref:`Auto Return <paypal/auto-return>` must also be enabled.
+.. tip::
+   PayPal **recommends** using **IPN**. For more information on **PDT** and **IPN**, click `here. <https://developer.paypal.com/api/nvp-soap/ipn/IPNPDTAnAlternativetoIPN/>`_
```

Comment:
The recommendation doesn't apply to the API Odoo integrates with and is bad practice anyway for integrators like Odoo databases. We should make it clear that all of Auto return, PDT and IPN **must** be enabled for Odoo to work properly.
```suggestion
```